crates/common/tedge_config/Cargo.toml

```diff
@@ -14,5 +14,5 @@ toml = "0.5"
 url = "2.2"
 
 [dev-dependencies]
-assert_matches = "1.4"
```

Comment:
It's not related to this PR, however, I found it coincidentally. All other crates are using `1.5`, only this crate uses `1.4`.